.generate/gen-all.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+#  Copyright 2019 The Operator-SDK Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+set -o errexit
+set -o pipefail
+
+# Turn colors in this script off by setting the NO_COLOR variable in your
+# environment to any value:
+#
+# $ NO_COLOR=1 test.sh
+NO_COLOR=${NO_COLOR:-""}
+if [ -z "$NO_COLOR" ]; then
+  header=$'\e[1;33m'
+  reset=$'\e[0m'
+else
+  header=''
+  reset=''
+fi
+
+function header_text {
+  echo "$header$*$reset"
+}
+
+RELEASE_VERSION=v1.0.0
+
+function install_bin() {
+	local url="https://github.com/operator-framework/operator-sdk/releases/download/${2}/${1}-${2}-x86_64-linux-gnu"
+	curl -sSLo $1 $url
+	chmod +x $1
+	...
+}
+
+install_bin operator-sdk $RELEASE_VERSION
+install_bin helm-operator $RELEASE_VERSION
+install_bin ansible-operator $RELEASE_VERSION
```

Comment:
Why do we need to install helm/ansible bin?
IHMO we do not need to get/install the sdk version at all since it we can do manually and then just run the script. 
It would make this one easier. 

@estroz @joelanford @asmacdo @varshaprasad96 objections?